<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>60</y>
      <width>511</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="debug_t">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Debug-меню:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="id_t">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ID пользователя:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="id"/>
     </item>
     <item>
      <widget class="QLabel" name="entry_t">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Дата входа:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="entry"/>
     </item>
     <item>
      <widget class="QLabel" name="exit_t">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Дата выхода:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="exit"/>
     </item>
     <item>
      <widget class="QLabel" name="home_num_t">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Номер дома:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="home_num"/>
     </item>
     <item>
      <widget class="QPushButton" name="debug_button">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Зарегистрировать запись</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>671</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="entries_t">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Входы в дом</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="update_table">
       <property name="text">
        <string>Обновить таблицу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="entries_widget">
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Дата/время входа</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Дата/время выхода</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Номер дома</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="error_label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
